nifi-api/src/main/java/org/apache/nifi/authorization/resource/Authorizable.java

```diff
@@ -67,7 +69,8 @@ default boolean isAuthorized(Authorizer authorizer, RequestAction action, NiFiUs
      * @return is authorized
      */
     default AuthorizationResult checkAuthorization(Authorizer authorizer, RequestAction action, NiFiUser user, Map<String, String> resourceContext) {
-        // TODO - include user details context
+        final Map<String,String> userContext = new HashMap<>();
+        userContext.put(UserContextKeys.CLIENT_ADDRESS.name(), user.getClientAddress());
```

Comment:
I think we have some instances where the client address is not set on the user. We should probably not include the userContext in those cases.